BraveShared/de.lproj/BraveShared.strings

```diff
@@ -332,7 +332,7 @@
 "defaultBrowserCallout.introTertiaryText" = "Öffnen Sie die Einstellungen, tippen Sie auf „Standardbrowser“ und wählen Sie Brave aus.";
 
 /* Notification body to promote setting Brave app as default browser */
-"defaultBrowserCallout.notificationBody" = "Optimiert für iOS 14. Legen Sie Brave gleich als Ihren Standardbrowser fest.";
+"defaultBrowserCallout.notificationBody" = "Optimiert für iOS %@. Legen Sie Brave gleich als Ihren Standardbrowser fest.";
```

Comment:
These does not need to be changed, they get updated automatically, with a translations script.
But this can stay this way no problem